packages/server/src/routes/members/index.ts

```diff
@@ -777,14 +787,25 @@ export const createMember = (
           throw new AlreadyRequestedError();
         }
 
-        const { username, fullName, videoToken, inviteToken } = call.body;
+        const {
+          username,
+          fullName,
+          emailAddress,
+          videoToken,
+          inviteToken
+        } = call.body;
 
         if (!username) {
           throw new MissingParamsError(["username"]);
         }
         if (!fullName) {
           throw new MissingParamsError(["fullName"]);
         }
+        // TODO Enable this check once we're sure all clients have upgraded to request email on signup.
```

Comment:
While were at it probably better to just combine these checks into a parameter checking method:

```typescript
const requiredParams = { username, fullName, emailAddress };
const missingParams = Object.keys().find(key => !requiredParams(key));
if (missingParams.length !== 0) {
  throw new MissingParamsError(missingParams);
}
```